### DISCUSSION TITLE
请帮我拿回激活码

### DISCUSSION THREAD

**Discussion by xiaolin6456** [getlantern/lantern-forum-cn#737] [General]
您好我升级到桌面应用程序最新版，可以登陆了，但是不能登陆专业版
购买时邮箱xiaolin6456@sina.com,及付款凭证
![Uploading QQ图片20240727102816.png…]()

**Comment by MaxMKMax** [comment]
@xiaolin6456 您好. 请您重新提供我支付单据. 因为被屏蔽了. 看不到呢. 谢谢!

**Comment by xiaolin6456** [comment]
您好：支付单据详见附件
----- 原始邮件 -----
发件人：Mike K ***@***.***>
收件人：getlantern/lantern-forum-cn ***@***.***>
抄送人：xiaolin6456 ***@***.***>,  Mention ***@***.***>
主题：Re:_[getlantern/lantern-forum-cn]_请帮我拿回激活码_(Discussion_#737)
日期：2024年07月29日 16点09分


@xiaolin6456 您好. 请您重新提供我支付单据. 因为被屏蔽了. 看不到呢. 谢谢!
